File: week_07/day_32/warm-up/getSquares.js
const getSquares = arr => {
  const result = [];
  arr.forEach(square=>{
    if (Math.sqrt(square) == Math.round(Math.sqrt(square))) {
      if (result.indexOf(square) == -1) {
        result.push(square)
      }
    }
  })
  return result.sort(numberSort);
}

const numberSort = (a,b) => {
    return a - b;
};

getSquares([4, 1, 16, 1, 10, 35, 22])

File: week_07/day_32/warm-up/get_squares.rb
def get_squares(arr)
  result = []
  arr.each do |square|
    if Math.sqrt(square) == Math.sqrt(square).round
      result.push(square)
    end
  end
  result.sort.uniq
end

puts get_squares([4, 1, 16, 1, 10, 35, 22])
